.github/workflows/on-config-pullrequest-dockerfile.yml

```diff
@@ -43,6 +43,9 @@ jobs:
         with:
           dockerfile: ${{ matrix.dockerfile.path }}
           failure-threshold: error
+          # DL3022 warning: `COPY --from` should reference a previously defined `FROM` alias
+          # ... this rule does not work well when the "from" is an additional _context_.
+          ignore: DL-3022
```

Comment:
Addresses this unhelpful warning:

![image](https://github.com/user-attachments/assets/1a6ee2cb-265f-448e-9d22-ab447f2a215c)